src/pages/index.js

```diff
@@ -32,6 +52,43 @@ const addNewCardButton = document.querySelector(".profile__add-button");
 const addCardContainer = addCardModal.querySelector("#add-card-container");
 const addCardSubmitButton = addCardModal.querySelector(".modal__button");
 
+/* -------------------------------------------------------------------------- */
+/*              variables that belong to the confirm delete popup             */
+/* -------------------------------------------------------------------------- */
+
+const confirmDeleteModal = document.querySelector("#confirm-delete-modal");
+const confirmDeleteModalContainer =
+  confirmDeleteModal.querySelector(".modal__container");
+const confirmDeleteModalForm = confirmDeleteModal.querySelector(".modal__form");
+const confirmDeleteModalCloseButton =
+  confirmDeleteModal.querySelector(".modal__close");
+const confirmDeleteModalSubmit =
+  confirmDeleteModal.querySelector(".modal__button");
+
+/* -------------------------------------------------------------------------- */
+/*              variables that belong to the change avatar popup              */
+/* -------------------------------------------------------------------------- */
+const changeAvatarModal = document.querySelector("#change-avatar-modal");
+const changeAvatarModalContainer = changeAvatarModal.querySelector(
+  "#change-avatar-container"
+);
+const changeAvatarModalForm = changeAvatarModal.querySelector(
+  "#change-avatar-form"
+);
+const changeAvatarModalCloseButton =
+  changeAvatarModal.querySelector(".modal__close");
+const chagneAvatarModalSubmitButton =
+  changeAvatarModal.querySelector(".modal__button");
+
+/* -------------------------------------------------------------------------- */
+/*                              card template div                             */
+/* -------------------------------------------------------------------------- */
+
+const cardSelector = "#card-template";
+const cardListSelector = ".cards__list";
```

Comment:
From the 8th sprint all constants should be in file `constants.js` in folder `utils` to clean up the main file.

Reply:
in this comment are you talking about the const cardSelector and cardlistselector being in the constant.js file?

Reply:
because i have the card constants placed in a constant.js in the utils folder since sprint 7

Reply:
> in this comment are you talking about the const cardSelector and cardlistselector being in the constant.js file?

Yes, cardSelector and cardlistselector 

and `validationSettings` as well (it's a constant object with selectors)

Reply:
Ohhh i see okay ill get work on that right now thank you again